features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin_new/source/src/app/components/ApplicationSettings/ListApplications.jsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import API from 'AppData/api';
+import { useIntl, FormattedMessage } from 'react-intl';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import HelpBase from 'AppComponents/AdminPages/Addons/HelpBase';
+import ListBase from 'AppComponents/AdminPages/Addons/ListBase';
+import EditApplication from 'AppComponents/ApplicationSettings/EditApplication';
+import DescriptionIcon from '@material-ui/icons/Description';
+import Link from '@material-ui/core/Link';
+import Configurations from 'Config';
+import EditIcon from '@material-ui/icons/Edit';
+
+/**
+ * Render a list
+ * @returns {JSX} Header AppBar components.
+ */
+export default function ListApplications() {
+    const intl = useIntl();
+    const [applicationList, setApplicationList] = useState([]);
+    /**
+    * API call to get application list
+    * @returns {Promise}.
+    */
+    function apiCall() {
+        return new Promise((resolve, reject) => {
+            const restApi = new API();
+            restApi
```

Comment:
Same here

Reply:
Do you mean the variable, `const restApi = new API();` (Line 48)?

Should we make it as follows?

```
(new API()).
    .getApplicationList()
    .....
```